.eleventy.js

```diff
@@ -89,6 +89,16 @@ module.exports = (config) => {
     throw new Error('readTemplate tag: unexpectedly reached end of template include directories')
   })
 
+  config.addShortcode('readReactExample', (componentName, exampleName) => {
+    try {
+      const examplePath = require.resolve(`@moduk/frontend/src/react/${componentName}/__examples__/${exampleName}.tsx`)
+      const code = readFileSync(examplePath, 'utf8')
+      return prettier.format(code, { parser: 'typescript' })
```

Comment:
We currently have a 120 max line length for TypeScript, which is a bit long in the code box – so am putting the code through Prettier here to get it down to 80.